src/TDockWidget.h

```diff
@@ -0,0 +1,46 @@
+#ifndef MUDLET_TDOCKWIDGET_H
+#define MUDLET_TDOCKWIDGET_H
+/***************************************************************************
+ *                                                                         *
```

Comment:
Heh, well in my head it was only a tiny bit of code I cobbled together from examples and code bits in the sources already.   
I will probably make more changes to TDockWidget when I add window layout code, so I suppose I'll add my info then.  :)